site/content/installation/ngf-images/pulling-ngf-image.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Pushing an NGINX Plus image for NGINX Gateway Fabric to a private registry
```

Comment:
Looking at the rendered docs, these two titles for the new docs are a bit long. I think for the titles let's just say `NGINX Plus image` instead of `NGINX Plus image for NGINX Gateway Fabric`. The description can still include it, but the titles are a bit long.